openpype/hosts/houdini/api/lib.py

```diff
@@ -1054,3 +1054,72 @@ def add_self_publish_button(node):
     template = node.parmTemplateGroup()
     template.insertBefore((0,), button_parm)
     node.setParmTemplateGroup(template)
+
+
+def getSceneViewer():
+    """
+    return an instance of a visible viewport.
+    There may be many, some could be closed, any visible are current
+    """
+    panes = hou.ui.paneTabs()
+    panes = [x for x in panes if x.type() == hou.paneTabType.SceneViewer]
+    panes = sorted(panes, key=lambda x: x.isCurrentTab())
+    if panes:
+        return panes[-1]
+    else:
+        return None
+
+
+def sceneview_snapshoot(
+        filename="$HIP/thumbnails/$HIPNAME.$F4.jpg",
+        fstart=None,
+        fend=None):
```

Comment:
Maybe it would worth to have option to pass in `scene_view` object instead of using `getSceneViewer`. So caller can tell which scene view should be actually used?